frontend/src/pages/landingPage/landingPage.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../components/Buttons/Buttons";
+@import "../../components/Basics/breakpoints";
+
+.image-parent {
+  z-index: 1;
+  margin-top: -10%;
+  @include media-max("smtablet") {
+    margin-top: 0;
+  }
+
+  position: relative;
+  background-image: url("../../assets/images/svgs/group-background-pattern.svg");
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 100%;
+}
+
+.group-image {
+  width: 100%;
+}
+
+.svg-group-layer {
+  margin: 0 12%;
+}
+
+.mission-section {
+  padding-top: 8%;
+  margin-top: -12%;
+  position: relative;
+  background-color: #ffefdb;
+  height: 420px;
+}
+
+.down-arrow {
+  width: 24px;
+
+  @include media-min("smtablet") {
+    display: none;
+  }
+
+  margin-bottom: 30px;
+}
+
+.mission-content {
+  margin: auto;
+  max-width: 800px;
+}
+
+.top-content {
+  max-width: 705px;
+  margin: auto;
+  padding-left: 50px;
+  padding-right: 50px;
+  margin-top: 80px;
+}
+
+// media query to change the default size would be easier this way
+
+// @include media-min("smtablet") {
+//   @include btn-size("lg");
+// }
+
+// code below isnt working working with Button component as it did with native button element
+
+.action-btn {
+  @include button-responsive("smtablet", "md");
+  @include button-responsive("lgtablet", "lg");
+  @include button-responsive("laptop", "lg");
+  @include button-responsive("desktop", "lg");
+}
```

Comment:
The screen sizes are not ordered from largest to smallest, so laptop lg will override the smaller size. This is what the current logic is:

If size is smtablet or smaller, use medium.
If size is lgtablet or smaller, use large.
If size is laptop or smaller, use large.
If size is desktop, use large.

Therefore, if the screen size is smtablet, it is smaller than laptop, so large is used. To get the behavior you want, you need to reorder the logic.

If size is desktop, use large.
If size is laptop or smaller, use large.
If size is lgtablet or smaller, use large.
If size is smtablet or smaller, use medium.

Since smtablet is smtablet or smaller, it will use medium.

The Design System Helper explains [this somewhat](https://aveline-art.github.io/CivicTechJobs-Documentation/developer/design-system/#responsive-mixins), although I might edit it a little so that it better applies to components with a default built in.